s3tests/functional/__init__.py

```diff
@@ -1,21 +1,20 @@
-from __future__ import print_function
 import sys
-import ConfigParser
+import configparser
```

Comment:
these can work in 2 and 3 if we use something like from six.moves import configparser